Makefile

```diff
@@ -9,6 +9,8 @@
 
 project_name := astrodynamics
 project_version := $(shell git describe --tags --always)
+python_version := 3.8
```

Comment:
I'd use sth like `jupyter_python_version` and same for the shared object. Basically, at this level of definition,  `project_name` and `project_version` are targeting the entire project (including the build of multiple python versions distribution) so I'd make the name of the var more specific as it's used only for running the test environment in jupyter

Reply:
I agree

Reply:
fixed